css/index.css

```diff
@@ -1,3 +1,396 @@
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+    max-width: 880px;
+    margin: auto;
+}
+
+@media( max-width: 800px ) {
+    .container {
+        max-width: 800px;
+        margin: 0px 20px;
+    }
+}
+
+@media( max-width: 500px ) {
+    .container {
+        max-width: 500px;
+        margin: 0px;
+    }
+}
+
+@media( max-width: 500px ) {
+    .container .main-content {
+        max-width: 500px;
+        margin: 20px;
+    }
+}
+
+@media( max-width: 500px ) {
+    .container .contact{
+        max-width: 500px;
+        margin: 50px 20px 20px 20px;
+    }
+}
+
+/* Header Styles */
+
+header {
+    width: 100%;
+    display: flex;
+}
+
+@media( max-width: 800px ) {
+    header {
+        flex-direction: column-reverse;
+    }
+}
+
+header nav {
+    margin-top: 40px; 
+    width: 610px;
+    justify-content: space-between;
+    display: flex;
+}
+
+@media( max-width: 800px ) {
+    header nav {
+        width: 100%;
+    }
+}
+
+@media( max-width: 500px ) {
+    header nav {
+        flex-direction: column;
+        align-items: center;
+        margin-top: 20px;
+        border-bottom: 1px solid gray;
+    }
+}
+
+header nav a {
+    text-decoration: none;
+    color: #b0b0b0;
+}
+
+@media( max-width: 500px ) {
+    header nav a {       
+        width: 100%;
+        padding: 20px;
+        border-top: 1px solid gray;
+        font-size: 20px;
+        text-align: center;
+    }
+}
+
+header nav a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+header .logo {
+    margin: 28px 0 0 91px;
+}
+
+@media( max-width: 800px ) {
+    header .logo {
+        width: 180px;
+        margin-left: 290px;
```

Comment:
I would pop off this margin-left here and use margin: auto or margin: 20px auto (to give top and bottom some room), this will automatically center the logo regardless of the width of the page past the break point

Reply:
I had tried using margin auto and the logo didn't respond to it at all. I'll play around with it after the project today.